test/fermionoperators.jl

```diff
@@ -31,6 +31,9 @@ const symmetries = (Trivial, U1Irrep)
 
         @test f_hop(sym) ≈ f_plus_f_min(sym) - f_min_f_plus(sym)
     end
+
+    @test_broken f⁻f⁻(U1Irrep)
+    @test_broken f⁺f⁺(U1Irrep)
```

Comment:
```suggestion
    @test_throws ArgumentError f⁻f⁻(U1Irrep)
    @test_throws ArgumentError f⁺f⁺(U1Irrep)
```

We want to consider this as a succeeded test if it throws, not as a broken one :)